src/shared/components/data-table/components/main/components/table-header/TableHeader.scss

```diff
@@ -0,0 +1,69 @@
+@import 'src/styles/common';
+
+.header {
+  height: 30px;
+  margin-top: 0px;
+  z-index: 1;
```

Comment:
Why is the z-index necessary here?

Reply:
For some reason the eye icon overlaps the header if I don't specify z-index. I'm not sure why.

<img width="129" alt="Screenshot 2022-08-18 at 12 39 44" src="https://user-images.githubusercontent.com/3085815/185386209-9e11d5c0-8803-48a0-bf77-6394b48284ae.png">